lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -216,6 +219,25 @@ export abstract class CacheManager implements ICacheManager {
         credential: ValidCredentialType
     ): string;
 
+    /**
+     * Read the user-supplied cache into memory, if applicable
+     * @param serializableCache 
+     * @param persistencePlugin 
+     */
+    async readUserDefinedCacheIntoMemory(
+        serializableCache: ISerializableTokenCache | null,
+        persistencePlugin: ICachePlugin | null
+    ): Promise<void> {
+        if (serializableCache && persistencePlugin) {
+            const cacheContext = new TokenCacheContext(
+                serializableCache,
+                true,
+            );
+
+            await persistencePlugin.beforeCacheAccess(cacheContext);
```

Comment:
The pattern is: 
```
try { 
     beforeCacheAccess
} 
finally { 
     afterCacheAccess 
}
```



`AfterCacheAccess` needs to be implemented as well, to allow implemeters to make changes.